cirq-core/cirq/ops/gateset.py

```diff
@@ -201,12 +203,18 @@ class Gateset:
     validation purposes.
     """
 
+    @_compat.deprecated_parameter(
+        deadline='v0.16',
+        fix='Add a global phase gate to the Gateset',
```

Comment:
Please make the `fix` message more descriptive, i.e. something like "To accept global phase gates, add `cirq.GlobalPhaseGate` to the list of `*gates` passed to the constructor. By default, global phase gates will not be accepted by the gateset".